client/post-editor/editor-visibility/index.jsx

```diff
@@ -158,21 +162,23 @@ export default React.createClass( {
 	},
 
 	updateVisibility( event ) {
-		var defaultVisibility, newVisibility, postEdits;
-
-		defaultVisibility = 'draft' === this.props.status ? 'draft' : 'publish';
-
-		postEdits = { status: this.props.savedStatus && 'private' !== this.props.savedStatus ? this.props.savedStatus : defaultVisibility };
-
-		newVisibility = event.target.value;
+		const { siteId, postId } = this.props;
+		const defaultVisibility = 'draft' === this.props.status ? 'draft' : 'publish';
+		const postEdits = {
+			status: this.props.savedStatus && 'private' !== this.props.savedStatus
+				? this.props.savedStatus
+				: defaultVisibility
+		};
+		const newVisibility = event.target.value;
+		let reduxPostEdits = false;
```

Comment:
This would work just as well (and arguably more semantically meaningful) not assigning a value:

```js
let reduxPostEdits;
```